.github/ISSUE_TEMPLATE/card_author_bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Card Author Bug Report
+about: Create a bug report for problems authoring Adaptive Crds
+title: "[Authoring] Bug Title Here"
+labels: Bug Triage-Needed Authoring
+assignees: ''
+---
+
+# Target Application
+
+Please list the application(s) that the card you are authoring is targeting (ex: Microsoft Teams, Outlook, etc)
+
+# Problem Description
+
+Please enter a dscription of the issue. If you just have a question, please post [on StackOverflow instead](https://stackoverflow.com/questions/tagged/adaptive-cards)
+
+# Expected Outcome
+
+Please describe what outcome you were expecting
+
+# Actual Outcome
+
+Please describe what was the actual outcome
+
+# Repro Steps
+
+Please enter the steps to reproduce the issue
+
+# Version of SDK
```

Comment:
People won't be able to provide that information for this type of issue in most cases

Reply:
Ah right, this should be Schema Version instead